index.html

```diff
@@ -11105,19 +11098,23 @@ <h2>Element displayedness</h2>
  considered visible if any part of it is drawn on the canvas within
  the boundaries of the viewport.
 
-<p>The <dfn data-lt="element displayedness|not displayed">element displayed</dfn> algorithm
- is a boolean state where <code>true</code> signifies that the element is displayed
- and <code>false</code> signifies that the element is not displayed.
- To compute the state on <var>element</var>,
- invoke the <a>Function.[[\Call]]</a>(<a><code>null</code></a>, <var>element</var>,
- <code>false</code>),
- with <a><code>bot.dom.isShown</code></a> as the this value.
- If doing so does not produce an error,
- return the return value from this function call.
- Otherwise return an <a>error</a> with <a>error code</a> <a>unknown error</a>.
-
-<p>This function is typically exposed to <code>GET</code> requests
- with a <a>URI Template</a> of
+<p>The <dfn>element displayed state</dfn> is a boolean representing
+whether an element is currently visible.
+
+<p>To get the <a>element displayed state</a> using
+the <a><code>bot.dom.isShown</code></a> Selenium atoms,
+given <var>element</var>:
+ <ol>
+  <li><p>Let <var>result</var> be
+  <a>Function.[[\Call]]</a>(<a><code>null</code></a>, <var>element</var>,
+  <code>false</code>), with <a><code>bot.dom.isShown</code></a> as the
```

Comment:
I wonder what the `false` argument stands for here. Nowadays the atoms get called with just the element.